source/facter/1.6/custom_facts.markdown

```diff
@@ -18,32 +11,32 @@ Sometimes you need to be able to write conditional expressions
 based on site-specific data that just isn't available via Facter,
 or perhaps you'd like to include it in a template.
 
-Since you can't include arbitrary ruby code in your manifests,
+Since you can't include arbitrary Ruby code in your manifests,
 the best solution is to add a new fact to Facter. These additional facts
 can then be distributed to Puppet clients and are available for use
 in manifests and templates, just like any other fact would be.
 
 ## The Concept
 
-You can add new facts by writing snippets of ruby code on the
-Puppet master. Puppet will then use [Plugins in Modules](/guides/plugins_in_modules.html)
+You can add new facts by writing snippets of Ruby code on the
+Puppet master. Puppet then uses [Plugins in Modules](/guides/plugins_in_modules.html)
 to distribute the facts to the client.
 
 ## Loading Custom Facts
 
 Facter offers a few methods of loading facts:
 
- * $LOAD\_PATH, or the ruby library load path
- * The environment variable 'FACTERLIB'
- * Facts distributed using pluginsync
+* `$LOAD\_PATH`, or the Ruby library load path
```

Comment:
If you wrap it in backticks, you don't need the backslash escape on the underscore. (Actually, on our MD renderer this never needed it.)